### ISSUE TITLE
Pre-commit doesn't find ktlint file

### ISSUE THREAD

**Issue by Kolyall** [pinterest/ktlint#634]
the file `ktlint` is located at `/home/nick/workspace/MyProject/ktlint`
in pre commit :
```
git diff --name-only --cached --relative | grep '\.kt[s"]\?$' | xargs ktlint --relative .
```
But if I make commit, the fail message is:
```
Commit failed with error
0 files committed, 1 file failed to commit: MyCommitMessage
xargs: ktlint: No such file or directory
```

But when I change pre commit file to full path to ktlint file:
```
git diff --name-only --cached --relative | grep '\.kt[s"]\?$' | xargs /home/nick/workspace/MyProject/ktlint --relative .
```
Then commit work correctly. Why the error occurs in first case?

**Comment by Kolyall** [comment]
Found issue, on Linux OS the line should be with `./ktlint` instead of `ktlint`
Fix in in your` --install-git-pre-commit-hook` command

**Comment by gotev** [comment]
Same happened on macOS using SourceTree. Solved it by adding:

```bash
export PATH=/usr/local/bin:$PATH
```

Resulting `pre-commit`:
```bash
#!/bin/sh
# https://github.com/shyiko/ktlint pre-commit hook
export PATH=/usr/local/bin:$PATH
git diff --name-only --cached --relative | grep '\.kt[s"]\?$' | xargs ktlint --relative .
if [ $? -ne 0 ]; then exit 1; fi
```

**Comment by Tapchicoma** [comment]
I think idea behind the hook is that `ktlint` executable should be available system-wide via placing it in some path defined in `PATH` env variable.

Enforcing `ktlint` executable to be always in the project root dir is also not nice.
